src/components/listings/MasonryListing.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import { View, Dimensions, ImageSourcePropType } from 'react-native'
+
+import MasonryListItem from '@/components/listings/MasonryListItem'
+
+interface MasonryListingProps {
+  images: ImageSourcePropType[]
+}
+
+function MasonryListing(props: MasonryListingProps) {
+  const { width } = Dimensions.get('window')
```

Comment:
should this be using the window dimensions or should we ref the root View in this file and reference that for it's width instead?